src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media/Given_ImageBrushWriteableBitmap.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UI.RuntimeTests.Extensions;
+using Uno.UI.RuntimeTests.Helpers;
+using Windows.Foundation.Metadata;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media.Imaging;
+using static Private.Infrastructure.TestServices;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Media
+{
+	[TestClass]
+	[RunsOnUIThread]
+	public partial class ImageBrush_WriteableBitmap
+	{
+		[TestMethod]
+		public async Task When_UseWriteableBitmapAsImageBrushSource()
```

Comment:
```suggestion
		public void When_UseWriteableBitmapAsImageBrushSource()
```

Reply:
Hello Youssef! Thank you for the review! The problem if I make that change is that Takescreenshot need that await.  var result = await RawBitmap.TakeScreenshot(imageBrush), becomes an error if I make that change.

Reply:
Is there a way to only change it for .Net6 mobile?

Reply:
I haven't seen the `await` when I added the comment. Sorry.

Regarding CI compile error, it will be fixed if the `#if !_MACOS__` is removed and a conditional `[Ignore]` is added.